### DISCUSSION TITLE
Mobile selector

### DISCUSSION THREAD

**Discussion by LyzardKing** [orchidjs/tom-select#247] [Q&A]
Hi. I wanted to know if there is a way to set the mobile selector style, which opens a "native" modal with the items in the selection.
In bootstrap-select it is possible with:

```
if (/Android|webOS|iPhone|iPad|iPod|BlackBerry/i.test(navigator.userAgent)) {
    $('.selectpicker').selectpicker('mobile');
}
```

**Comment by oyejorge** [comment]
That's interesting though I'm not sure how we'd do it with TomSelect at the moment
